customersupport/templates/template.html

```diff
@@ -41,9 +41,9 @@
         </div>
         <div id="navbar" class="collapse navbar-collapse">
             <ul class="nav navbar-nav">
-                <li class="active"><a href="{{ url_for("new_call") }}">New Call</a></li>
+                <li><a href="{{ url_for("new_call") }}">New Call</a></li>
```

Comment:
I added this back in (with an even better way than that StackOverflow answer) in commit 8ad6df399fc997c0f040cc18c68246c7f01cf789.